lib/helpers/merge-exports.js

```diff
@@ -0,0 +1,11 @@
+"use strict"
+
+function mergeModules (target, module) {
+  for (var key in module) {
+    if (Object.prototype.hasOwnProperty.call(module, key)) {
```

Comment:
fwiw for robustness i'd still suggest using `hasown` or `var hasOwn = Function.call.bind(Object.prototype.hasOwnProperty);`